lib/cell-manager.coffee

```diff
@@ -37,3 +27,39 @@ module.exports = CellManager =
         console.log 'CellManager: Cell [start, end]:', [start, end], 'row:', row
 
         return [start, end]
+
+
+    getBreakpoints: ->
+        editor = atom.workspace.getActiveTextEditor()
+        buffer = editor.getBuffer()
+        breakpoints = [0]
+
+        regex = @getRegex editor, 'g'
+        if regex?
+            buffer.scan regex, ({range}) ->
+                breakpoints.push range.start.row
+
+        breakpoints.push editor.getLastBufferRow()
+
+        console.log 'CellManager: Breakpoints:', breakpoints
+
+        return breakpoints
+
+
+    getRegex: (editor, option = '') ->
```

Comment:
No need to act on this comment, but probably a `getRegexString` would've been a more versatile function.